app/onboarding/page.tsx

```diff
@@ -0,0 +1,179 @@
+const states = ['Tennessee', 'Missouri'];
+const gardenTypes = ['Individual', 'Community', 'School'];
+const plotOptions = [
+  { label: 'yes', value: true },
+  { label: 'no', value: false },
+];
+//Interfaces and props to avoid typ errors on Lint
+interface StateSelectionProps {
+  selectedState: string;
+  setSelectedState: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface GardenTypeSelectionProps {
+  selectedGardenType: string;
+  setSelectedGardenType: React.Dispatch<React.SetStateAction<string>>;
+}
+
+interface PlotSelectionProps {
+  selectedPlot: boolean | null;
+  setSelectedPlot: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+// Select State
+const StateSelection: React.FC<StateSelectionProps> = ({
+  selectedState,
+  setSelectedState,
+}) => {
+  return (
+    <div>
+      <h2>Which state do you live in?</h2>
+      <select
+        value={selectedState}
+        onChange={e => setSelectedState(e.target.value)}
+      >
+        <option value="" disabled>
+          Select a state
+        </option>
+        {states.map(state => (
+          <option key={state} value={state}>
+            {state}
+          </option>
+        ))}
+      </select>
+    </div>
+  );
+};
+
+// Step 2: Select garden type
+
+const GardenTypeSelection: React.FC<GardenTypeSelectionProps> = ({
+  selectedGardenType,
+  setSelectedGardenType,
+}) => {
+  return (
+    <div>
+      <h2>What type of garden do you want to create?</h2>
+      {gardenTypes.map(type => (
+        <label key={type}>
+          <input
+            type="radio"
+            name="gardenType"
+            value={type}
+            checked={selectedGardenType === type}
+            onChange={e => setSelectedGardenType(e.target.value)}
+          />
+          {type}
+        </label>
+      ))}
+    </div>
+  );
+};
+
+// Step 3: Do you have a plot?
+const PlotSelection: React.FC<PlotSelectionProps> = ({
+  selectedPlot,
+  setSelectedPlot,
+}) => {
+  return (
+    <div>
+      <h2>Do you already have a plot?</h2>
+      {plotOptions.map(option => (
+        <label key={option.label}>
+          <input
+            type="radio"
+            name="plot"
+            value={String(option.value)}
+            checked={selectedPlot === option.value}
+            onChange={() => setSelectedPlot(option.value)}
+          />
+          {option.label}
+        </label>
+      ))}
+    </div>
+  );
+};
+
+// Main Onboarding Component
+const OnboardingFlow = () => {
+  const [step, setStep] = useState(1);
+  const [selectedState, setSelectedState] = useState<string>('');
+  const [selectedGardenType, setSelectedGardenType] = useState<string>('');
+  const [selectedPlot, setSelectedPlot] = useState<boolean>(false);
+
+  const handleNext = () => {
+    setStep(step + 1);
+  };
+
+  const handleBack = () => {
+    setStep(step - 1);
+  };
+
+  const handleSubmit = async () => {
+    const profile: Profile = {
+      user_id: id,
+      state: selectedState,
+      email: '',
```

Comment:
ooo i'm so sorry i'm realizing we will not need the email field in the `profiles` table or the `Profile` type since the email will be available from the users table (and therefore auth context)
also, we decided to remove `phone_num` from Profiles too.
so let's remove email and phone_num from the Profile type and edit this!

Reply:
Noted!